source/extensions/common/wasm/context.cc

```diff
@@ -1087,6 +1087,44 @@ const Network::Connection* Context::getConnection() const {
   return nullptr;
 }
 
+std::string spanToString(const absl::Span<const std::string>& span) {
```

Comment:
Can we do [`absl::StrJoin(span, ",")`](https://github.com/abseil/abseil-cpp/blob/d43b7997c0ca0f3312a51d1057fb73cfe934703b/absl/strings/str_join.h#L172-L173)?